libs/qlik/src/utils/vl-qlik.types.ts

```diff
@@ -0,0 +1,63 @@
+import { VlSelect } from '@domg-wc/elements';
+
+export type Filter = {
```

Comment:
persoonlijk gebruik ik nooit 'type' maar altijd 'interface' - alhoewel ze grotendeels gelijkaardig zijn
typescript zelf heeft ook een voorkeur voor interfaces: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces